### PR TITLE
Hf token

### PR DESCRIPTION
**Pull request by Sukh-P**
# Pull Request

## Description

Adds a secret and sets the value of the secret as an environment variable for poetry python test workflow, tested here https://github.com/openclimatefix/india-forecast-app/actions/runs/11107579398/job/30859075552?pr=103

### CODE REVIEW

**Review by peterdudfield — APPROVED** [review]

File: .github/workflows/poetry-python-test.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ on:
         required: false
         type: string
         default: 'unassigned'
+    secrets:
+      HUGGINGFACE_TOKEN:
+        required: false
+        description: 'Token for private HF repo'
```
Comment (peterdudfield) [inline .github/workflows/poetry-python-test.yml:12]: actually should you add a default here

Comment (peterdudfield) [reply .github/workflows/poetry-python-test.yml:12]: default: 'unassigned'

Comment (Sukh-P) [reply .github/workflows/poetry-python-test.yml:14]: Hmmm seems not to like them https://github.com/openclimatefix/india-forecast-app/actions/runs/11107579398

Comment (peterdudfield) [reply .github/workflows/poetry-python-test.yml:14]: maybe just remove them then ....

**Pull request merged**